lib/course.rb

```diff
@@ -1 +1,22 @@
+class Course
+  attr_reader :name,
+              :capacity
+  attr_accessor :students
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @students = []
+  end
 
+  def full?
+    if @students.count < 2
```

Comment:
```suggestion
    if @students.count < @capacity
```
It would be more dynamic to use the course's `@capacity`. Some courses might allow more than 2 students to enroll which would break this method.